src/views/BasicPage.vue

```diff
@@ -14,14 +14,51 @@
 				required: true
 			}
 		},
+		data() {
```

Comment:
as before, consistency
```suggestion

		data() {
```